Source/Download.swift

```diff
@@ -141,7 +152,7 @@ extension Request {
         var downloadProgress: ((Int64, Int64, Int64) -> Void)?
 
         var resumeData: NSData?
-        override var data: NSData? { return resumeData }
+        override var data: NSData? { return self.resumeData }
```

Comment:
It's certainly not necessary, but brings it inline with all other properties used throughout Alamofire. 

Previously, there were some instances where we were using `self` on properties, and others where we were not. IMO, it is MUCH easier to not make mistakes if you specify `self` on properties. Otherwise naming collisions can actually cause some un-anticipated problems. Especially when your function parameter names overlap with your property names which is a pretty common use case.

Therefore, a few commits ago, I audited the codebase for properties without `self` and added it to have the same convention throughout Alamofire. When working through this PR, I realized I missed one and made the change. While I agree with you that it's not necessary, I would certainly vote to keep the convention.

Thoughts?